superset-frontend/src/components/Icons/index.tsx

```diff
@@ -65,6 +65,7 @@ const IconFileNames = [
   'dataset_virtual',
   'download',
   'drag',
+  'drag_icon',
```

Comment:
I'm not sure `drag_icon` is the best name, all declarations here are icons. What's the difference between the drag icons? Maybe we can come up with a better name.

Reply:
Nahh, I just forgot to delete that (added for test purposes). Good eye as always 😄